distributions/openhab/src/main/resources/bin/backup

```diff
@@ -55,9 +55,15 @@ setup(){
 
   fileList="$OPENHAB_RUNTIME/bin/userdata_sysfiles.lst"
 
+  ## Parse arguments
+  if [ "$1" = "--full" ] || [ "$2" = "--full" ]; then
```

Comment:
I think it would be safer to request that the "--full" parameter is done before the listed directory. This way, `--full` will always be stored in `$1`. Inside this function, you can add the shift command, now `$1` thereafter will always contain the full path as expected by the rest of the script.

Reply:
Hmmm... I actually put it as the first argument first... But then I looked at the other arguments to openhab-cli and noticed that the only other command that has a "--"-argument (start --debug) has it last so that felt more consistent. But I can change it back again ;-)